### DISCUSSION TITLE
What should I do

### DISCUSSION THREAD

**Discussion by Harrypotter098** [The-Art-of-Hacking/h4cker#64] [Q&A] [answered]
Hey I am in 12th now 

I want to get into ethical Hacking

So what should I do now what kind of exam should I take 

Can some one help me out

**Comment by YaneyLily** [comment]
感谢您的来信！
      ——杨莉（Yaney）

**Comment by Jace-07** [comment]
look up a couple of projects and practice, see what works for you and maybe you'll have enough data to make an informed decision....

**Comment by jabrielcjones** [comment, marked as answer]
Check out training services like O'Reilly Media, aCloudGuru, Cybary, and Udemy. On those sites search for "ethical hacking" and "penetration testing".

Also, check out Hack the Box and Try Hack Me. Both other penetration testing training and labs.

If you want certs, check out https://www.coursera.org/articles/popular-cybersecurity-certifications

**Comment by itbuddies** [comment]
**Here are some exams that you can take to get into ethical hacking:**

- **Certified Ethical Hacker (CEH):** This is a popular certification that is offered by EC-Council. The exam covers a wide range of topics in ethical hacking, such as network security, web security, and malware analysis.
- **Offensive Security Certified Professional (OSCP):** This is an advanced certification that is offered by Offensive Security. The exam is more hands-on than the CEH exam and requires you to demonstrate your skills in attacking and defending systems.
- **CompTIA Security+:** This is a general security certification that is offered by CompTIA. The exam covers a wide range of topics in security, including network security, risk management, and incident response.

Which exam you take depends on your level of experience and your goals. If you are new to ethical hacking, I recommend starting with the CompTIA Security+ exam. Once you have some experience, you can then consider taking the CEH or OSCP exam and also go with an [ethical hacking tutorial](https://www.tutorialsfreak.com/ethical-hacking-tutorial) which really helps you to clear your examinations easily.
